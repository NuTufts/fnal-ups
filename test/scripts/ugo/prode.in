Product:          ProdE
Flavor:           IRIX+5.3
                  IRIX+5
                  IRIX
                  NULL
Qualifiers:       cat
                  
DB:               /thunder
                  /lightening
Chains:           current
ugo_v 0 and UPS_VERBOSE 0
