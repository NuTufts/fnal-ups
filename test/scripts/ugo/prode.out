Product:          ProdE
Flavor:           IRIX+5.3
                  IRIX+5
                  IRIX
                  NULL
Qualifiers:       cat
                  
DB:               /home/t2/fagan/ups_database
Chains:           current
